openfisca_france/model/prelevements_obligatoires/impot_revenu/ir.py

```diff
@@ -167,6 +166,26 @@ def function(self, simulation, period):
         return period, self.sum_by_entity(enfant_a_charge.array * not_(garde_alternee.array))
 
 
+# class nombre_enfants_a_charge_menage(PersonToEntityColumn):
+#     entity_class = Menages
+#     label = u"Nombre d'enfants à charge  non mariés, de moins de 18 ans au 1er janvier de l'année de perception des" \
+#         u" revenus, ou nés durant la même année ou handicapés quel que soit leur âge"
+#     operation = 'add'
+#     variable = enfant_a_charge
+#
+#
+# class enfant_a_charge_invalide(Variable):
+#     column = BoolCol
+#     entity_class = Individus
+#     label = u"Enfant à charge titulaire de la carte d'invalidité"
+#
+#     def function(self, simulation, period):
+#         period = period.this_year
+#         enfant_a_charge = simulation.compute('enfant_a_charge', period)
+#         garde_alternee = simulation.compute('garde_alternee', period)
+#         return period, self.sum_by_entity(enfant_a_charge.array * not_(garde_alternee.array))
+
```

Comment:
Make sure to remove this commented code before merging.

If these variables are not useful anymore, please mention their deprecation on the CHANGELOG.
If they are just move somewhere else, just delete the code.